src/block/BlockBreakInfo.php

```diff
@@ -73,7 +73,7 @@ public static function instant(int $toolType = BlockToolType::NONE, int $toolHar
 		return new self(0.0, $toolType, $toolHarvestLevel, 0.0);
 	}
 
-	public static function indestructible(float $blastResistance = 18000000.0) : self{
+	public static function indestructible(float $blastResistance = 18000003.75) : self{
```

Comment:
???

Reply:
For some reason there is a mismatch in blast resistance between indestructible blocks. All new indestructible blocks added since 1.2+ use 18000003.75, but older ones (bedrock, portal frames) use 1800000.0